firmware/ec/platform/oc-sdr/schema/auto_schema.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+#ifndef _AUTO_SCHEMA_H
+#define _AUTO_SCHEMA_H
+
+#include "common/inc/global/Framework.h"
```

Comment:
Is it possible to generate this file as well ?

Reply:
It's possible to auto generate the header file, but the h-files could be anywhere in the directory structure, so we'd have to either move them to a common location, or provide the location in a separate specification, i.e.:
```
{
    "Framework" : "global",
    "ocmp_adt7481" : "ocmp_wrappers",
    ...
    ...
}
```
So my thought was that a separate specification for the headers would be more cumbersome than just updating this header by hand.